app/uk/gov/hmrc/accessibilitystatementfrontend/views/fragments/openbanking/TechnicalDetailsOpenBanking.scala.html

```diff
@@ -0,0 +1,69 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.accessibilitystatementfrontend.models.AccessibilityStatement
+@import uk.gov.hmrc.play.language.LanguageUtils
+@import uk.gov.hmrc.accessibilitystatementfrontend.models.dateToLocalDate
+
+@this(utils: LanguageUtils)
+
+@()(implicit messages: Messages, statement: AccessibilityStatement)
+
+@import statement._
+
+<h2 class="govuk-heading-l">@{messages("technical.heading")}</h2>
+<p class="govuk-body">@{messages("technical.commitment")}</p>
+
+@if(isFullyCompliant) {
+    <p class="govuk-body">@{messages("technical.compliance.full")} <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">@{messages("technical.compliance.guidelines")}</a>@{messages("general.fullstop")}</p>
+} else {
+    @if(isNonCompliant) {
+        <p class="govuk-body">@{messages("technical.compliance.noncompliant")} <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">@{messages("technical.compliance.guidelines")}</a>@{messages("technical.compliance.partial.reasons")}
+        </p>
+    } else {
+        @if(hasMilestones) {
+        <p class="govuk-body">@{messages("technical.compliance.partial")} <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">@{messages("technical.compliance.guidelines")}</a>@{messages("technical.compliance.partial.reasons")}</p>
+      } else {
+         <p class="govuk-body">@{messages("technical.compliance.partial")} <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">@{messages("technical.compliance.guidelines")}</a>@{messages("general.fullstop")}</p>
+        }
+    }
+}
+
+@technicalNonCompliance = {
+    <h3 class="govuk-heading-m">@{messages("technical.non-compliance.header")}</h3>
+    <p class="govuk-body">@{messages("technical.non-compliance.introduction")} @if(displayAutomatedTestingOnlyContent) {@automatedTestingDetails.getOrElse("")}</p>
+    <h4 class="govuk-heading-s">@{messages("technical.non-compliance.accessibility")}</h4>
+}
+
+@if(hasMilestones && isPartiallyCompliant) {
+    @technicalNonCompliance
+    @for(milestone <- milestones.get) {
+        <p class="govuk-body">@{milestone.description} @{messages("technical.non-compliance.milestone.fix-by", utils.Dates.formatDate(dateToLocalDate(milestone.date)))}</p>
+    }
+
+    @if(displayAutomatedTestingOnlyContent) {
+        <p class="govuk-body">@{messages("technical.non-compliance.automated.test-by")}</p>
+    }
+}
+
+@if(isNonCompliant) {
+    @technicalNonCompliance
```

Comment:
@JoPintoPaul should the three lines under this be removed, seems like it might display it twice - might be worth creating some screenshots of the output of the report in different states to share with dias for sign-off if we do it like this (guess partially compliant with no issues, and partially compliant with some issues)

Reply:
noticed there's two keys in the "how accessible is this service" template that have .openbanking suffix but none in here, I wonder if for this template we could update the standard template instead - seems like if our validations are only changing for statements with type open banking to allow a partially compliant statement with no milestones then changing the standard template might have same effect without the cost of future maintenance of having an extra template - unless I have misread somewhere